.github/workflows/test-next.yml

```diff
@@ -0,0 +1,118 @@
+name: "[CI] Test Next"
+
+on:
+    push: # Only run on merges into main that modify files under pyscriptjs/ and examples/
+        branches:
+            - next
+        paths:
```

Comment:
Are we sure these paths are correct?

Reply:
I copied the build workflow one so maybe?

Reply:
hmm, the `next` branch has this new folder also, see https://github.com/pyscript/pyscript/tree/next
it's called `[pyscript.core](https://github.com/pyscript/pyscript/tree/next/pyscript.core)`

Looping in @WebReflection here...

Reply:
core needs a story a part for integration and CI (currently integration is manual check) but `pyscriptjs` should always be tested as non breaking, imho, whenever any file changes in there ... does it make sense?

Reply:
OK, pyscript core has a 100% code coverage so maybe that's also important moving forward, but integration (automated) is missing.

Reply:
I think we are GTG @FabioRosado